assets/blocks/quiz/question-block/question-settings.js

```diff
@@ -0,0 +1,46 @@
+import { InspectorControls } from '@wordpress/block-editor';
+import { PanelBody } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { QuestionGradeSettings } from './settings';
+
+/**
+ * Question block settings controls.
+ *
+ * @param {Object}     props                    Block props.
+ * @param {Function[]} props.controls           Additional setting controls.
+ * @param {Object}     props.attributes         Block attributes.
+ * @param {Object}     props.attributes.options Block options attribute.
+ * @param {Function}   props.setAttributes      Update block attributes.
+ */
+const QuestionSettings = ( {
+	controls = [],
+	attributes: { options = {} },
+	setAttributes,
+} ) => {
+	const setOptions = ( next ) =>
+		setAttributes( { options: { ...options, ...next } } );
+
+	controls = [ QuestionGradeSettings, ...controls ];
```

Comment:
I wonder if would make sense to have the `QuestionGradeSettings` in the `assets/blocks/quiz/answer-blocks/index.js` too. Even it being a standard for all the options, I think it would be nice to have this responsibility in a single place.

Reply:
Yeah that makes sense. Moved it in 5addc5c